tmp/app/assets/stylesheets/refills/_vertical-tabs.scss

```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
```

Comment:
Now may not be the time to adjust, but I believe this px -> em mixin calculator is being deprecated in Bourbon.